qa/common/common.sh

```diff
@@ -726,3 +726,29 @@ echo "Result: OK"
 EOF
     _run_test_script_on_node $TESTSCRIPT $STORAGENODE
 }
+
+function configure_OSD_to_filestore {
```

Comment:
this function appears to change *all* the OSDs to filestore? Maybe the function name could be changed to reflect that, e.g `configure_all_OSDs_to_filestore`